<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChronoServer</class>
 <widget class="QMainWindow" name="ChronoServer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>279</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChronoServer</string>
  </property>
  <widget class="QWidget" name="m_central_widget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Competitor</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="m_lbl_photo">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="../../../common/icons/chrono.qrc">:/common/nobody.png</pixmap>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label_2">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>First name :</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="m_lbl_first_name">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_3">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Last name :</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="m_lbl_last_name">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_4">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Round :</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="m_lbl_round">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Status</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="m_lbl_status">
           <property name="font">
            <font>
             <weight>75</weight>
             <italic>true</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Waiting ...</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_pb_arm">
           <property name="text">
            <string>Arm</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>279</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="m_lbl_time">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>60</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>30</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string>border: 4px solid black;</string>
        </property>
        <property name="text">
         <string>00:00:000</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>278</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>537</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="m_action_preferences"/>
    <addaction name="separator"/>
    <addaction name="m_action_quit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="m_action_about"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="m_status_bar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="m_action_listen"/>
   <addaction name="m_action_test"/>
   <addaction name="m_action_quit"/>
  </widget>
  <action name="m_action_preferences">
   <property name="icon">
    <iconset resource="../../../common/icons/chrono.qrc">
     <normaloff>:/common/configure.png</normaloff>:/common/configure.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Preferences ...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="m_action_quit">
   <property name="icon">
    <iconset resource="../../../common/icons/chrono.qrc">
     <normaloff>:/common/exit.png</normaloff>:/common/exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="m_action_test">
   <property name="icon">
    <iconset resource="../../../common/icons/chrono.qrc">
     <normaloff>:/common/tester.png</normaloff>:/common/tester.png</iconset>
   </property>
   <property name="text">
    <string>Test serial</string>
   </property>
   <property name="toolTip">
    <string>Test serial</string>
   </property>
  </action>
  <action name="m_action_about">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="m_action_listen">
   <property name="icon">
    <iconset resource="../../../common/icons/chrono.qrc">
     <normaloff>:/server/bind.png</normaloff>:/server/bind.png</iconset>
   </property>
   <property name="text">
    <string>Listen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../../common/icons/chrono.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>m_action_quit</sender>
   <signal>activated()</signal>
   <receiver>ChronoServer</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
